CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
> I don't know which keyboard you are but if I type once the backtick touch, I directly have a backtick.
German keyboard layout on Mac. When I type one tick and then any char, the char got the acute. Two ticks stops the acute building

Back to topic: I searched the docs in this repo. We use two ticks ~1700 times, one tick ~400 times...